src/halmos/__main__.py

```diff
@@ -958,6 +908,7 @@ def run_parallel(run_args: RunArgs) -> list[TestResult]:
             setup_config,
             with_devdoc(args, fun_info.sig, run_args.contract_json),
             libs,
+            run_args.build_out_map,
```

Comment:
the serialization of this additional item seems to be causing the [performance overhead](https://github.com/a16z/halmos/actions/runs/10662927742) (6m vs 3m) in the test-parallel mode.

Reply:
to address the performance issue, we can consider removing unnecessary fields from the build json output before serialization.

Reply:
we discussed and decided to deprecate the test-parallel mode once the portfolio solving feature is ready.